### DISCUSSION TITLE
Can I use this dlt-viewer within a QML Application?

### DISCUSSION THREAD

**Discussion by vamsi0303** [COVESA/dlt-viewer#597] [Q&A]
I was trying to add this dlt-viewer within my own QML project. is it possible to that? is yes how I achieve it ?

**Comment by vifactor** [comment]
dlt-viewer is an application, not a library to be used in another application. please be more specific with your question.

**Comment by alexmucde** [comment]
Possibly you can use the qdlt library, but i am not familiar with QML.
